### PR TITLE
track2 option in clicore

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

**Review by changlong-liu — APPROVED** [review]

File: src/entry.ts [hunk 1]
```diff
@@ -129,10 +129,10 @@ function processGenerationOption(session: Session<CodeModel>) {
     let isSdkNeeded = !cliCore;
     const generateSdk = AzConfiguration.getValue(CodeGenConstants.generateSDK);
     isSdkNeeded = isNullOrUndefined(generateSdk) ? isSdkNeeded : generateSdk === GenerateSdk.Yes;
-    const compatibleLevel =
-        AzConfiguration.getValue(CodeGenConstants.compatibleLevel) || cliCore
-            ? CompatibleLevel.Track1
-            : CompatibleLevel.Track2;
+    let compatibleLevel = cliCore ? CompatibleLevel.Track1 : CompatibleLevel.Track2;
```
Comment (lirenhe) [inline src/entry.ts:132]: Could you rename it to compatibilityLevel?

File: test/scenario-test.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import * as path from 'path';
 import { CodeGenConstants, CompatibleLevel, GenerateSdk, TargetMode } from '../src/utils/models';
 import { copyRecursiveSync, deleteFolderRecursive, isNullOrUndefined } from '../src/utils/helper';
 import * as sourceMapSupport from 'source-map-support';
+import { Test } from 'mocha';
```
Comment (lirenhe) [inline test/scenario-test.ts:9]: I think we no longer use mocha for testing.

Comment (qiaozha) [reply test/scenario-test.ts:9]: Jest pr hasn't been merged yet. This is for purely bugfixing

File: src/entry.ts (hunk 1 above)
Comment (qiaozha) [reply src/entry.ts:132]: Why? Just think it may cause confusion for our customer who already had tried --compatible-level before.

**Pull request merged**